R/utilities-parameter.R

```diff
@@ -1,3 +1,43 @@
+
+#' Retrieve all parameters of a container (simulation or container instance) matching the given path criteria

```

Comment:
@PavelBal Do you know of any good reference to create Roxygen docs? Somehow I am a bit struggling with the keywords

This is what I have at the mometn
![image](https://user-images.githubusercontent.com/1041237/61912218-830fe680-af07-11e9-9622-3326cf4ccbe3.png)


If I do not use the don't run key, the examples are being run when the package is built which does not work. Anyways I'll create an issue for that